uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -621,7 +643,6 @@
             var leaveRequest = LeaveRequestInstance.init(mockData.findBy('status_id', status));
 
             leaveRequest.contact_id = CRM.vars.leaveAndAbsences.contactId.toString();
-            role = 'manager';
```

Comment:
Why this is removed?

Reply:
It is not removed, it is moved exactly at the place where it says "this is manager". 

https://github.com/compucorp/civihr/pull/2706/files#diff-e66ee11e23aa834592aec412809fbf34R596